transaction-service/src/main/java/pl/adamboguszewski/transaction/service/infrastructure/exception/TransactionExceptionHandler.java

```diff
@@ -42,7 +43,7 @@
     @ExceptionHandler(IllegalPaymentTypeArgumentException.class)
     public ResponseEntity<CreateTransactionResponse> handle(IllegalPaymentTypeArgumentException exception) {
         log.info("Payment type " + exception.getPaymentType() + " not recognized.");
```

Comment:
Also as you can see now every of our exception handler method is doing same thing, maybe we should just catch some abstract exception class huh